### PR TITLE
Development

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
Added HTML and CSS for Landing Page

### CODE REVIEW

**Review by julialoeschel — COMMENTED** [review]
nice. Das das h2 fehlt hatte mich verwirrt, aber du hast es bis zum Ende durchgezogen. Ich hatte trotzdem verstanden dass das h2 nicht fehlen darf. 
Ansonsten mega nice, vorallem dass der Answer Button auch Answert :)

File: index.html [hunk 1]
```diff
@@ -5,7 +5,94 @@
...
+      crossorigin="anonymous"
+    />
     <title>Quizzzle</title>
   </head>
-  <body></body>
+  <body>
+    <nav>
+      <dl class="menu">
+        <dt>home</dt>
+        <dd class="active"><a href="#"></a><i class="fas fa-home"></i></a></dd>
```
Comment (badabam) [inline index.html:20]: Please consider an `ul` with `li`s  instead. Also: the anchor closes twice in this line. And it needs some text for accessibility.

File: index.html [hunk 2]
```diff
@@ -5,7 +5,94 @@
...
+      </dl>
+    </nav>
+
+    <header>Quizzzle</header>
+
+    <main>
+      <ul class="card-container">
+        <li class="question-card">
+          <div class="question-container">
+            <h3>Question</h3>
```
[the remark this answers is not in the document]
Comment (badabam) [reply index.html:36]: Julia is right.

File: style/style.css [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+nav {
+  position: fixed;
+  z-index: 10;
+  bottom: 0px;
```
Comment (badabam) [inline style/style.css:26]:
```suggestion
  bottom: 0;
```

File: style/style.css [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+  display: grid;
+  grid-row: 1fr;
+  gap: 25px;
+  row-gap: 15px 15px;
+  padding: 15px;
+}
+
+.question-card {
+  position: relative;
+  box-shadow: rgba(0, 0, 12, 0.24) 0px 3px 5px;
```
Comment (badabam) [inline style/style.css:80]: Check everywhere for `0px` -> `0`

File: style/style.css [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+  border: solid 3px #999;
+  border-radius: 999px;
+  color: #999;
+  font-weight: bold;
+}
+
+.bookmark {
+  position: absolute;
+  top: -10px;
+  right: -5px;
```
Comment (badabam) [inline style/style.css:162]: Correct!

**Review by badabam — CHANGES_REQUESTED** [review]
Well done! I found some things for you to consider. Keep on the good work!

File: index.html (hunk 1 above)
Comment (F-Kirchhoff) [reply index.html:20]: thats why I did the dl so there would be a title for the screen reader. How could I do it correctly?

File: style/style.css [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 10px;
+  font-size: 1rem;
+}
+
+.tag-container li {
+  padding: 5px 8px;
+  border: solid 3px #999;
+  border-radius: 999px;
```
[the remark this answers is not in the document]
Comment (F-Kirchhoff) [reply style/style.css:154]: the border radius? thats my hack to ensure that there the rounding is not stretched when the div isnt a perect square.

File: index.html (hunk 1 above)
Comment (badabam) [reply index.html]: The text needs to be in the anchor. Do it like we did it together in my example.
```html
<nav>
  <ul>
    <li><a href="#">Home</a></li>
   <!-- ... -->
 </ul>
</nav>
```

**Pull request merged**